### PR TITLE
Use new service worker routing

### LINKED ISSUE #1966: Use new service worker route
**Is your feature request related to a problem? Please describe.**
Following https://github.com/bbc/simorgh/pull/1965, we should use the new service worker route for the ws frontpages

| service | path |
|:--------|:-----|
| news | /service/articles/sw.js |
| persian | /service/sw.js |
| igbo | /service/sw.js |
| yoruba | /service/sw.js |
| pidgin | /service/sw.js |

https://github.com/bbc/simorgh/blob/latest/src/client.js#L21-L23

When doing this PLEASE do not import the service config into the client.js file, this will cause big problems that will immediately need to be refactored for bundle splitting

**Describe the solution option 1**
Do a `if (service === 'news') {} else {}` to decide which service worker URL to use 

**Describe the solution option 2**
Move the service worker install into the react application and pass `serviceWorkerPath` from the service context

**Testing notes**
Regression tests routes still work.

Dev insight: Unit tests

### PR DESCRIPTION
**Pull request by EinsteinNjoroge**
Resolves #1966

**Overall change:** _Configure frontpages to use the new service worker routes defined in service config._

**Code changes:**

- _Move service worker installation to a separate component inside `App`_
- _Add service worker paths to service config_

---

- [X] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval
- [ ] I have followed [the merging checklist](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/MERGE_PROCESS.md) and this is ready to merge.

### CODE REVIEW

**Review by dr3 — APPROVED** [review]

File: src/app/containers/ServiceWorker/index.test.jsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  });
+
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  describe('on production environment', () => {
+    it('should be installed', async () => {
+      process.env.NODE_ENV = 'production';
+      mount(
```
Comment (ghost) [inline src/app/containers/ServiceWorker/index.test.jsx]: You should unmount this after the assertion

File: src/app/containers/ServiceWorker/index.test.jsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+};
+
+describe('Service Worker', () => {
+  beforeEach(() => {
+    global.navigator.serviceWorker = {
+      register: jest.fn(),
+    };
+  });
+
+  afterEach(() => {
```
Comment (ghost) [inline src/app/containers/ServiceWorker/index.test.jsx:19]: There is no need to clear mocks

File: src/app/containers/ServiceWorker/index.test.jsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import ServiceWorkerContainer from './index';
+import { ServiceContext } from '../../contexts/ServiceContext';
+
+const mockRootElement = <div />;
+document.getElementById = jest.fn().mockReturnValue(mockRootElement);
```
Comment (ghost) [inline src/app/containers/ServiceWorker/index.test.jsx]: We don't need to mock this

**Review by ghost — CHANGES_REQUESTED** [review]

**Review by dr3 — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Comment by j-pendlebury** [comment]
@EinsteinCarrey can you take a look at your snapshots tests? Jenkins and Travis seem to be failing

**Comment by david-boydell** [comment]
Hi @EinsteinCarrey,

I've checked this and the following routes return a 404: 

http://localhost:7080/pidgin/service/sw.js
http://localhost:7080/service/sw.js

I'm not entirely sure what I need to do to get this to work? Any ideas?

**Comment by EinsteinNjoroge** [comment]
@david-boydell  try replacing `service` with the service name ie. 
[http://localhost:7080/**news/articles**/sw.js](http://localhost:7080/news/articles/sw.js)
[http://localhost:7080/**pidgin**/sw.js](http://localhost:7080/pidgin/sw.js)

**Comment by david-boydell** [comment]
Ah, cool, thanks @EinsteinCarrey, this is ready for merge.

**Pull request merged**